dbfit-java/build.gradle

```diff
@@ -134,16 +204,39 @@ subprojects {
             }
     }
 
-    // to be overriden when needed
-    ext.connectionPropertiesFiles = []
-
     ext.copyConnectionProperties = {
         destination ->
             project.connectionPropertiesFiles.each { file ->
                 copyProperties new File(project.projectDir, file), destination
             }
     }
 
+    ext.execSqlScriptTestDatabase = {
+        scriptFile ->
```

Comment:
> If we're not fussed which we use right now then we can just log another issue to standardise later.

OK. The changes related to this PR we may directly format as we want. And whatever else needs reformatting - could be done later.